\documentclass{resume} % Use the custom resume.cls style
\name{Kim, Dongwook} % Your name
\address{GÃ©nopode 1015.3, University of Lausanne \\ 1015 Lausanne, Switzerland} % Your address
%\address{Born 29.10.1996 in Seoul. Citizen of Republic of Korea. }
%\address{+82~$\cdot$~2~$\cdot$~880~$\cdot$~4438}
\address{\Letter~dongwook.kim@unil.ch $\vee$ endix1029@gmail.com} % Your phone number and email
\usepackage[left=0.75in,top=0.6in,right=0.75in,bottom=0.6in]{geometry} % Document margins
%\usepackage{fullpage}
\usepackage{setspace}  % controllable line spacing 
\usepackage{graphicx}
\usepackage{eso-pic}
\usepackage[utf8]{inputenc}
\usepackage{comment}

%\usepackage{picins}    % um das Foto in die rechte obere Ecke zu bekommen
\usepackage{eurosym}
\usepackage{color}
\usepackage{pdfpages}

\definecolor{urlcol}{rgb}{0,0,0.7}
\definecolor{highlight}{rgb}{0.5,0,0}
\usepackage[hidelinks]{hyperref}
\usepackage[normalem]{ulem}

%Custom commands
\renewcommand{\url}[1]{\uline{\href{#1}{#1}}}
\newcommand{\uhref}[2]{\href{#1}{\uline{#2}}}
\newcommand{\doi}[1]{\uhref{https://doi.org/#1}{#1}}
\newcommand{\piinfo}[2]{\hfill\textcolor{black!70}{\fontsize{10pt}{11.5pt}\selectfont #1: \textit{#2}}}

%\usepackage[sfdefault]{roboto}
%\usepackage[scaled]{helvet}
\usepackage{libertinus}
\renewcommand{\familydefault}{\sfdefault}

\newcommand{\subheading}[1]{%
  \vspace{0.5mm}%
  {\fontsize{11.5pt}{13pt}\selectfont\bfseries #1}%
  \vspace{1mm}%
}

\pagestyle{empty}
\oddsidemargin  -0.2cm
\evensidemargin -0.2cm
\textwidth      16.5cm
\headheight     0.0cm
\topmargin      -1.8cm
\textheight     26cm
\parindent      0cm
%\renewcommand{\familydefault}{\sfdefault}
\setlength{\parskip}{5pt}

\renewcommand\refname{}
\renewcommand{\contentsname}{\LARGE Curriculum Vitae\\[20mm]{\Large Content\\[0mm]}}
% \providecommand{\href}[2]{#2}                          %generates the href macro if needed

\newenvironment{packed_itemize}{
\begin{itemize}
  \setlength{\itemsep}{3pt}
  \setlength{\parskip}{0pt}
  \setlength{\parsep}{0pt}
}{\end{itemize}}
 
\DeclareMathVersion{mymath}
\DeclareSymbolFont{myletters}{OML}{zplm}{m}{it}
\SetSymbolFont{letters}{mymath}{OML}{zplm}{m}{it}
\DeclareSymbolFont{myoperators}{OT1}{pplx}{m}{n}
\SetSymbolFont{operators}{mymath}{OT1}{pplx}{m}{n}

\mathversion{mymath}

\begin{document}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\setcounter{tocdepth}{1}

\pagestyle{plain}
\setcounter{page}{1}

\setlength{\oddsidemargin}{-0.2cm}
\setlength{\evensidemargin}{-0.2cm}
\setlength{\linewidth}{16.5cm}
\setlength{\textheight}{26cm}
\setlength{\tabcolsep}{0cm}

\vspace{3mm}
\begin{rSection}{Curriculum Vitae}
\vspace{1mm}

\subheading{Education}

\begin{tabular}{p{3.6cm}p{13.0cm}}
03/2019 - 02/2025 & Ph.D. in Bioinformatics at Seoul National University
\\[1.mm]
03/2019 - 02/2022 & Completed Master's and Doctor's Integration Course in Bioinformatics at Seoul National University
\\[1.mm]
03/2015 - 02/2019 & B.S. in Biological Sciences, minor in Computer Sciences at Seoul National University, \textit{cum laude}
\\[1mm]
\end{tabular}
\vspace{1mm}

\subheading{Research experience}

\begin{tabular}{p{3.6cm}p{13.0cm}}
05/2025 - & Postdoctoral fellow at the Comparative Genomics Group, University of Lausanne.\\
& \piinfo{PI}{Prof. Christophe Dessimoz \& Dr. Natasha Glover} \\[1.mm]
03/2025 - 04/2025 & Postdoctoral fellow at the Steinegger Lab., Seoul National University. \\
& \piinfo{PI}{Prof. Martin Steinegger} \\[1.mm]
%03/2022 - 02/2025 & Expert Research Personnel at the College of Natural Sciences, Seoul National University. \textit{full-time}
%\\[1.mm]
12/2021 - 02/2025 & Ph.D. candidate at the Steinegger Lab., Seoul National University. \\
& \piinfo{Advisor}{Prof. Martin Steinegger} \\[1.mm]
03/2019 - 11/2021 & Ph.D. candidate at the Laboratory of Evolutionary Bioinformatics, Seoul National University. \\
& \piinfo{Advisor}{Prof. Jongsik Chun} \\[1.mm]
06/2017 - 02/2019 & Intern at the Laboratory of Evolutionary Bioinformatics, Seoul National University
\\[1.mm]
06/2014 - 07/2014 & Visiting student at the Neurovascular Research Laboratory, Department of Radiology, Massachusetts General Hospital, Harvard Medical School
\\[1.mm]
03/2013 - 02/2014 & Visiting student at Hyeon Soo Kim Lab., Department of Anatomy, College of Medicine, Korea University
\\[1.mm]
\end{tabular}
\vspace{1mm}
\end{rSection}

\newpage

\begin{rSection}{Talks, Posters, and Publications}
\vspace{1mm}

\subheading{Talks}

\begin{tabular}{p{2.5cm}p{14.1cm}}
02/2025 & The 21\textsuperscript{st} KOGO Winter Symposium, Hongcheon, Korea, Introducing Unicore: A Scalable and Accurate Method for Structural Core Gene Phylogenetics
\\[1mm]
07/2023 & ISMB/ECCB 2023, Lyon, France, UFCG: database of universal fungal core genes and pipeline for genome-wide phylogenetic analysis of fungi
\\[1mm]
11/2021 & IMBG Young Scientist Symposium, Seoul, Korea, EzAAI - A High-Throughput Pipeline for Prokaryotic AAI Calculations
\\[1mm]
\end{tabular}
\vspace{1mm}

\subheading{Posters}

\begin{tabular}{p{2.5cm}p{14.1cm}}
04/2025 & RECOMB 2025, Seoul, Korea, Scalable and accurate structural core gene phylogeny with Unicore
\\[1mm]
02/2025 & The 21\textsuperscript{st} KOGO Winter Symposium, Hongcheon, Korea, Introducing Unicore: A Scalable and Accurate Method for Structural Core Gene Phylogenetics
\\[1mm]
07/2024 & SMBE 2024, Puerto Vallarta, Mexico, Unicore Enables Ultra-fast and Accurate Phylogenetic Reconstruction with Structural Core Genes
\\[1mm]
07/2023 & ISMB/ECCB 2023, Lyon, France, UFCG: database of universal fungal core genes and pipeline for genome-wide phylogenetic analysis of fungi
\\[1mm]
06/2020 & ASM Microbe Online, Virtual, UUCGf: Whole-Genome Profiling Pipeline of Fungi with Fungal Core Gene Set for High-Resolution Phylogenetics
\end{tabular}
\vspace{4mm}

\subheading{Peer-reviewed publications}\\[-26mm]

\begin{thebibliography}{10}

\vspace{8mm}

\bibitem{kim2025unicore}
{\bf Kim D.}, Park S. and Steinegger M. (2025),
Unicore enables scalable and accurate phylogenetic reconstruction with structural core genes, {\em Genome Biol. Evol.}, doi: \doi{10.1093/gbe/evaf109}

\bibitem{park2025structural}
Park J., Lee G., Han Y., {\bf Kim D.}, Heo K. et al. (2025),
Structural basis of the catalytic and allosteric mechanism of bacterial acetyltransferase PatZ, {\em Proc. Natl. Acad. Sci. U. S. A.}, doi: \doi{10.1073/pnas.2419096122}

\bibitem{kim2023ufcg}
{\bf Kim D.}, Gilchrist C.L.M., Chun J. and Steinegger M. (2023),
UFCG: database of universal fungal core genes and pipeline for genome-wide phylogenetic analysis of fungi, {\em Nucleic Acids Res.}, doi: \doi{10.1093/nar/gkac894}

\bibitem{kim2021ubcg2}
Kim J., Na S., {\bf Kim D.} and Chun J.  (2022), 
UBCG2: Up-to-date bacterial core genes and pipeline for phylogenomic analysis, {\em J. Microbiol.}, doi: \doi{10.1007/s12275-021-1231-4}

\bibitem{kim2021introducing}
{\bf Kim D.}, Park S. and Chun J. (2021),
Introducing EzAAI: a pipeline for high throughput calculations of prokaryotic average amino acid identity, {\em J. Microbiol.}, doi: \doi{10.1007/s12275-021-1154-0}

\end{thebibliography}
\vspace{4mm}

\begin{comment}
\subheading{Preprints}\\[-26mm]

\begin{thebibliography}{10}

\vspace{8mm}

\end{thebibliography}
\vspace{4mm}
\end{comment}

\subheading{Public source codes}

\begin{tabular}{p{2.5cm}p{14.1cm}}
Unicore &  \url{https://github.com/steineggerlab/unicore} \\[1.mm]
UFCG  & \url{https://github.com/steineggerlab/ufcg} \\[1.mm]
EzAAI  & \url{https://github.com/endixk/ezaai} \\[1.mm]
\end{tabular}
\vspace{2mm}

\end{rSection}
\newpage

\begin{rSection}{Achievements and Qualifications}
\vspace{1mm}

\subheading{Awards, Fellowships and Achievements}\vspace{1mm}

\begin{tabular}{p{2.5cm}p{14.1cm}}
2025 & KOGO Young Scientist Award
\\[1.mm] 
2021 & IMBG Young Scientist Award
\\[1.mm] 
2017 & Samsung Convergence Software Course Scholarship, KRW 1,000,000
\\[1.mm] 
2015 & National Science \& Technology Scholarship from the National Research Foundation of Korea, KRW 24,033,000
\\[1.mm] 
\end{tabular}
\vspace{1.0mm}

\subheading{Certificates}\vspace{1mm}

\begin{tabular}{p{2.5cm}p{14.1cm}}
2019 & Advanced english proficiency, Test of English Proficiency (Score 492/600, percentile rank 94.65\%)
\\[1.mm] 
\end{tabular}
\vspace{1.0mm}

\subheading{Technical Strengths}\vspace{1mm}

\begin{tabular}{ p{2.5cm} >{}l @{\hspace{6ex}} l }
Programming & Java, Rust, Python, C/C\texttt{++}, Linux, Shell scripting \\[1.mm]
Databases & SQL, MySQL
\\[1.mm]
\end{tabular}
\vspace{1.0mm}

\subheading{Languages}\vspace{1mm}

\begin{tabular}{p{2.5cm}p{14.1cm}}
Korean & Native\\[1.mm]
English & Fluent\\[1.mm]
French & Beginner\\[1.mm]
\end{tabular}

\end{rSection}
\vspace{5mm}

\begin{rSection}{Teaching Experiences}
\vspace{1mm}

\begin{tabular}{p{2.5cm}p{14.1cm}}
2025/11 & Guest lecturer, Reviews in Quantitative Biology, University of Lausanne
\\[1mm]
2021/09 & Teaching assistant, Microbial Systematics, Seoul National University
\\[1mm]
2020/09 & Teaching assistant, Microbial Systematics, Seoul National University
\end{tabular}
\vspace{4mm}

\end{rSection}

\end{document}

